lib/releases.nix

```diff
@@ -0,0 +1,22 @@
+# nixpkgs/nixos release information
+rec {
+
+  # formatting releases
+  asString = {
+    name
+  , releaseDate
+  }: "${name} (release date ${releaseDate})";
+
+  next = nixos-17-first;
```

Comment:
not 17.03)? If it's 17.03, there is no deprecation period for users of stable channels.

When you update your stable channel you are expected to read the changelog and fix any evaluation errors that arise in one swoop. At least for renames it gives a nice evaluation error and will be documented in the changelog.